src/encryption/encryption.go

```diff
@@ -0,0 +1,96 @@
+package encryption
+
+import (
+	"keboola-as-code/src/state"
+	"keboola-as-code/src/utils"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/iancoleman/orderedmap"
+	"go.uber.org/zap"
+)
+
+func isKeyToEncrypt(key string) bool {
+	return strings.HasPrefix(key, "#")
+}
+
+func isEncrypted(value string) bool {
+	return regexp.MustCompile(`^KBC::(ProjectSecure|ComponentSecure|ConfigSecure)::.+$`).MatchString(value)
```

Comment:
podla [dokumentacie](https://developers.keboola.com/overview/encryption/) tam moze byt ProjectSecure ComponentSecure alebo ConfigSecure tak som to podobne vymenoval aj tu